index.bs

```diff
@@ -1147,6 +1147,13 @@ Meetings</h3>
 	about the date and location of a meeting.
 	Shorter notice for a meeting is allowed
 	provided that there are no objections from group participants.
+	For new proposed [=Working Groups=] or [=Interest Groups=]
+	that have not yet been chartered,
+	the minimum announcement period
+	starts on the date of the <a href=#cfp>Call for Participation</a>,
+	and meetings <em class=rfc2119>must not</em> be announced before the call has been issued.
```

Comment:
not sure if that rule is still being pursued, but an impact of this is that it's likely to create a 3 to 4 months period of the year that is bad to launch new groups due to TPAC scheduling: assuming an end-of October physical TPAC, groups needs to be launched before mid-August to be part of TPAC (a much thought-after goal for new groups), right at the end of one of the worst period to run an AC review (mid-July to mid-August due to northern hemisphere Summer vacation time). Given how challenging launching a group can be, I'm not sure this is an additional barrier that should be added

Reply:
ah. I wonder if we can be more relaxed about TPAC because, after all, everyone comes anyway, right?